modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -1732,10 +1744,15 @@ public Expression textTranslateAnchor() {
                 "symbol-spacing", getId());
         symbolizer.getOptions().put("repeat", String.valueOf(symbolSpacing));
 
-        // text max angle
-        // layer.getTextMaxAngle();
-        // symbolizer.getOptions().put("maxAngleDelta", "40");
-
+        // text max angle - only for line placement
+        // throw MBFormatException if point placement
+        if(labelPlacement instanceof LinePlacement){
+            // followLine will be true if line placement, it is an implied default of MBstyles.
+            symbolizer.getOptions().put("followLine", "true");
+            symbolizer.getOptions().put("maxAngleDelta", String.valueOf(getTextMaxAngle()));
+        } else if (hasTextMaxAngle()) {
+        	throw new MBFormatException("Property text-max-angle requires symbol-placement = line but symbol-placement = " + symbolPlacementVal + "\"");
```

Comment:
You've got a trailing escaped quote, and no opening quote here: `\"`. It should be removed.